packs/feats/torrential-backlash.json

```
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Requirements</strong> You are in a curse maelstrom state.</p>\n<hr />\n<p>Calling upon forbidden practices, you release the curse from within yourself, allowing it free rein to destroy everything around you. All creatures within your curse maelstrom emanation except you take [[/r (@actor.level)d6[void]]]{1d6 void damage for each level} you have, with a @Check[type:fortitude|dc:resolve(@actor.system.attributes.classOrSpellDC.value)|basic:true] save. Your curse maelstrom state then ends.</p>"
+            "value": "<p><strong>Requirements</strong> You are in a curse maelstrom state.</p><hr /><p>Calling upon forbidden practices, you release the curse from within yourself, allowing it free rein to destroy everything around you. All creatures within your curse maelstrom emanation except you take @Damage[(@actor.level)d6[void]]{1d6 void damage for each level} you have, with a @Check[fortitude|dc:resolve(@actor.system.attributes.classOrSpellDC.value)|basic] save. Your curse maelstrom state then ends.</p>"
```

```suggestion
            "value": "<p><strong>Requirements</strong> You are in a curse maelstrom state.</p><hr /><p>Calling upon forbidden practices, you release the curse from within yourself, allowing it free rein to destroy everything around you. All creatures within your curse maelstrom emanation except you take @Damage[(@actor.level)d6[void]]{1d6 void damage for each level} you have, with a @Check[type:fortitude|dc:resolve(@actor.system.attributes.classOrSpellDC.value)|basic:true] save. Your curse maelstrom state then ends.</p>"
```
Please don't remove those in the system itself. Until we change all of them over I'd like to keep the style consistent.

---

It's on my to-do list to move the system to a new standard when I rewrite the style guide this summer